sys/net/routing/aodvv2/reader.c

```diff
@@ -616,6 +618,7 @@ static enum rfc5444_result _cb_rerr_blocktlv_addresstlvs_okay(struct rfc5444_rea
 
     VDEBUG("%s()\n", __func__);
     VDEBUG("\tmessage type: %d\n", cont->type);
+    /* cppcheck-suppress Unused variable as VDEBUG is needed for better debugging output. */
```

Comment:
One small nitpick: As far as I checked, this suppression is actually not necessary, since this is the place where nbuf is actually used, *if* it is used at all. :)